package.json

```diff
@@ -31,6 +31,7 @@
     "bluebird": "3.4.6",
     "body-parser": "1.15.2",
     "bookshelf": "0.10.2",
+    "brute-knex": "git://github.com/cobbspur/brute-knex.git#master",
```

Comment:
Could you change this to pin to a branch or even better a commit? If you pin to master and then make more changes, the code will change between releases :)